.github/scripts/authors_in_cff.py

```diff
@@ -23,56 +26,66 @@ def get_pr_authors() -> set[str]:
     authors = {
         commit["author"]["login"] for commit in response.json() if commit["author"]
     }
-    return authors - set(EXCLUDED_USERS)
+
+    return sorted(authors - excluded_authors)
 
 
-def check_citation_file(authors: set[str]) -> tuple[bool, str | None]:
+def find_missing_github_usernames(authors: list[str]) -> list[str]:
     """Verify that all authors of a PR are included in :file:`CITATION.cff`."""
     with pathlib.Path("CITATION.cff").open() as file:
-        contents = file.read()
-        for author in authors:
-            if f"alias: {author}" not in contents:
-                return False, author
-    return True, None
+        lines = file.read()
+        return [
+            author
+            for author in authors
+            if f"alias: {author}" not in lines and f"- alias: {author}" not in lines
+        ]
 
 
 def main():
     """Check that all authors are included in CITATION.cff."""
     authors = get_pr_authors()
-    check_passed, missing_github_username = check_citation_file(authors)
-    if not check_passed:
-        branch_name = os.getenv("GITHUB_HEAD_REF")
+    missing_github_usernames = find_missing_github_usernames(authors)
+
+    if not missing_github_usernames:
+        msg = (
+            f"The authors of pull request {PR_NUMBER} for {REPO} are: "
+            f"{', '.join(sorted(authors))}. All authors are included "
+            "in CITATION.cff. ✔️"
+        )
+        logging.info(msg)
+        sys.exit(0)
+
+    branch = os.getenv("GITHUB_HEAD_REF")
+    username = missing_github_usernames[0]
+
+    instructions_to_add_author = f"""
+
+To ensure that you get credit for your contribution, please add the
+following authors to CITATION.cff: {", ".join(missing_github_usernames)!r}
 
-        error_message = f"""
-To ensure that you get credit for your contribution to PlasmaPy, please
-add {missing_github_username!r} as an author to CITATION.cff.
+This file can be edited at:
 
-The entry should be of the form:
+https://github.com/{username}/PlasmaPy/edit/{branch}/CITATION.cff
+
+Each entry should be of the form:
 
 - given-names: <given names>
   family-names: <family names>
   affiliation: <affiliation>
   orcid: https://orcid.org/<ORCiD number>
-  alias: {missing_github_username}
-  email: <email address>
```

Comment:
We don't have a need for people's emails, so we can remove this. Plus, very few people have added the email field to their `CITATION.cff` entry, except for the most frequent contributors.